### ISSUE TITLE
4 Types of Classification Tasks in Machine Learning

### ISSUE THREAD

**Issue by khuyentran1401** [khuyentran1401/machine-learning-articles#15]
## TL;DR
Overview about types of classification tasks and which algorithms to use with each task

### Article Link
https://machinelearningmastery.com/types-of-classification-in-machine-learning/

### Author
Jason Brownlee

## Key Takeaways
4 main type of classification tasks
* Binary Classification: 0 or 1
* Multi-Class Classification: model predicts the probability of an example belonging to each class label.
* Multi-Label Classification: classification tasks that have two or more class labels, where one or more class labels may be predicted for each example.
* Imbalanced Classification: classification tasks where the number of examples in each class is unequally distributed.
